tests/unit/auth.cpp

```diff
@@ -352,6 +381,23 @@ TEST_F(AuthWithStorage, UserPasswordCreation) {
     ASSERT_FALSE(auth->Authenticate("test", "456"));
     ASSERT_TRUE(auth->RemoveUser(user->username()));
   }
+
+  {
+    auto user =
+        auth->AddUser("test", "sha256:d74ff0ee8da3b9806b18c877dbf29bbde50b5bd8e4dad7a3a725000feb82e8f1" /* pass */);
```

Comment:
Does this mean `CREATE USER 'test' IDENTIFIED BY 'sha256:d74ff0ee8da3b9806b18c877dbf29bbde50b5bd8e4dad7a3a725000feb82e8f1';` would also work?

Reply:
Yes! 
I'll add some e2e tests as well